main/release/utils/github.py

```diff
@@ -62,19 +75,22 @@ def __fake_release_request(self) -> ReleaseRequest:
 
     @Dryable(logging_msg='{function}()')
     def revoke_release(self) -> None:
-        tag_name = self._get_release()["tag_name"]
+        release = self._get_release()
+        if release is None:
+            return
+        tag_name = release["tag_name"]
         headers = {'Authorization': f'token {self.token}'}
         payload = {'draft': True, 'tag_name': tag_name}
-        requests.patch(self._get_release()['url'], json=payload, headers=headers)
+        requests.patch(release['url'], json=payload, headers=headers)
         # Delete tag
         requests.delete(self._get_repository().get("git_refs_url").replace("{/sha}", f'/tags/{tag_name}'), headers=headers)
 
     @staticmethod
     def is_publish_to_binaries():
         return os.environ.get('INPUT_PUBLISH_TO_BINARIES', 'false').lower() == "true"
 
-    def _get_release(self) -> {}:
-        return self.event["release"]
+    def _get_release(self) -> Union[dict, None]:
```

Comment:
I'm aware of `dict | None` being preferred but the Dockerfile does not build for Python 3.10.